.teamcity/src/subprojects/build/core/JDKBuild.kt

```diff
@@ -27,6 +27,11 @@ class JDKBuild(
             defineTCPPortRange()
         }
 
+        script {
+            name = "Install pulseaudio"
```

Comment:
Let's clarify the purpose here. Not to get this deleted by someone on refactoring
```suggestion
            name = "Install pulseaudio for WebRTC tests"
```

Reply:
Or add a comment that `webrtc.rs` requires `pulseaudio`